### ISSUE TITLE
PDF Export writes messages in reverse order

### ISSUE THREAD

**Issue by tripleee** [thundernest/import-export-tools-ng#32]
It doesn't seem to matter how I arrange the message view -- the PDF export writes them from newest to oldest, rather than the opposite. This is easy to fix if you are moderately computer literate, but a weird quirk which should hopefully not be too hard to fix.

(Giving the messages a sane naming structure would also help. It seems that the message's index number within the folder is the _last_ component of the filename before the `.pdf` extension.)

**Comment by tripleee** [comment]
In case you are *not* moderately computer literate, try something like

```
for msg in *.pdf; do
    idx=${msg##*-}
    idx=${idx%.pdf}
    printf -v new "%05i-%s.pdf" "$idx" "${msg%-$idx.pdf}"
    mv "$msg" "$new"
done
```
This should work in Bash, i.e. on Mac and Linux and other U*x. If you are a Windows victim, I can only offer my condolences.

**Comment by cleidigh** [comment]
@tripleee 
Under the filename option section you can set 3 message items to be included in the filename.
You can move the index to the front of the filename in this fashion.  Is that a partial solution?
I am not sure I have the control the write order
@cleidigh

**Comment by tripleee** [comment]
Not really, though it'snice to have control over that of course. Does it make the message index zero-padded so the messages sort correctly out of the box?

**Comment by cleidigh** [comment]
@tripleee 
check out #75 
will the full formatting help in your issue?
